apps/studio/components/interfaces/Storage/AnalyticsBuckets/index.tsx

```diff
@@ -111,15 +105,20 @@ export const AnalyticsBuckets = () => {
                     </TableRow>
                   )}
                   {analyticsBuckets.map((bucket) => (
-                    <TableRow key={bucket.id}>
+                    <TableRow
+                      key={bucket.id}
+                      className="cursor-pointer h-16"
+                      onClick={(event) => {
+                        const url = `/project/${ref}/storage/analytics/buckets/${encodeURIComponent(bucket.id)}`
+                        if (event.metaKey) window.open(url, '_blank')
+                        else router.push(url)
+                      }}
+                    >
+                      <TableCell className="w-2 pr-1">
+                        <BucketIcon size={16} className="text-foreground-muted" />
```

Comment:
You're the expert here :D but IMO the bucket icon looks like a cross between a trash icon and a lock icon, it just doesn't make me think "oh right, that's a bucket!"

Reply:
Fair point! I will tinker and push updates in a separate PR.